README.md

```diff
@@ -1,8 +1,35 @@
 # Cozy Developer Guides
 
+## Triggers contained in this repository
+
+- `CompoundExchangeRate`: Triggers if a Compound market's `exchangeRateStored` drops between consecutive blocks
+- `YearnSharePrice`: Triggers if a Yearn vault's `pricePerShare` drops between consecutive blocks
+
+## Development and Deployment
+
+1. Install dependencies with `yarn`
+2. Compile contracts with `yarn compile`
+3. Run tests with `yarn test`
+
+If tests are passing, you can deploy trigger contracts with the following steps:
+
+1. Copy the file called `.env.example` and rename it to `.env`
+2. Fill in your `RPC_URL` (such as an Infura endpoint) and the `PRIVATE_KEY` which is the private key of the deployer account. `ETHERSCAN_API_KEY` is only needed for verifying contracts
+3. To deploy the Compound protection market against a local forked network, run `yarn run hardhat scripts/create-protection-market-compound.ts`. To deploy it against mainnet use `yarn run hardhat scripts/create-protection-market-yearn.ts --network mainnet`
```

Comment:
this should say `yarn hardhat run`, not `yarn run hardhat`

Reply:
Fixed in 58cfbc7f95a9f3bc9de9467eed37bb4311e0f78d